services/core-web/src/tests/components/modalContent/UpdateMineReportSubmissionStatusModal.spec.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import UpdateMineReportSubmissionStatusModal from "@/components/modalContent/UpdateMineReportSubmissionStatusModal";
+import * as MOCK from "@/tests/mocks/dataMocks";
+import { reduxForm, reducer as formReducer } from "redux-form";
+
+const mockStore = configureStore({
```

Comment:
We do already have the ReduxWrapper component which will configure the store, wrap with provider, etc, just needs that `form` on ln 22 here passed in as initialState.

Reply:
Updated